mite/logoutput.py

```diff
@@ -75,6 +79,10 @@ def print_output(self, t):
         dt = t - self._start_t
         self._resp_time_recent.sort()
         self._logger.info(f"Total> #Reqs:{self._req_total} #Errs:{self._error_total}")
+        # only output journey logs if the --journey_logging switch is set
+        if self._journey_logging:
+            for k, v in self._error_journeys.items():
+                self._logger.info(f"Total errors for {k} :{v}")
```

Comment:
```suggestion
        for k, v in self._error_journeys.items():
            self._logger.info(f"Total errors for {k} :{v}")
```

...and then this doesn't need to be conditional, if we are filling the dict conditionally (since in the journey_logging=False case the dict will never have any entries, so this loop will never fire)

Reply:
journey errors are always logged, however, displaying them is conditional, which i why i do it this way

Reply:
Ok.  I may not have fully grasped the importance of recording the errors vs displaying them, but I can well imagine why you might want to proceed in the way you have described, so I don't object to the way it is now by any means :+1: